### PR TITLE
Problem: DenomID in NFT module does not accept capital letters

### PR DESCRIPTION
**Pull request by devashishdxt**
Solution: Modified error message clarifying that DenomID only accepts lowecase letters. Fixes #542.

### CODE REVIEW

**Review by tomtau — COMMENTED** [review]
is that a problem -- or is it better for denoms to be lowercase only (to prevent confusion or similarly named denoms)?
and if so, whether the testnet should go with the current 2.0.0 pre-release or create a new release with this patch?

**Comment by leejw51crypto** [comment]
small123, SMALL123,
Small123 all will be different denomID, that can cause some confusion

**Comment by devashishdxt** [comment]
> is that a problem -- or is it better for denoms to be lowercase only (to prevent confusion or similarly named denoms)?
> and if so, whether the testnet should go with the current 2.0.0 pre-release or create a new release with this patch?

We can use small letters only. In that case we can modify the error message clarifying this?

**Comment by devashishdxt** [comment]
> yes, modifying error message seems better solution

Changed

**Review by leejw51crypto — APPROVED** [review]
lgtm,

small spelling issue in nft-id

File: x/nft/types/validation.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func ValidateTokenID(tokenID string) error {
 		return sdkerrors.Wrapf(ErrInvalidTokenID, "the length of nft id(%s) only accepts value [%d, %d]", tokenID, MinDenomLen, MaxDenomLen)
 	}
 	if !IsBeginWithAlpha(tokenID) || !IsAlphaNumeric(tokenID) {
-		return sdkerrors.Wrapf(ErrInvalidTokenID, "nft id(%s) only accepts alphanumeric characters, and begin with an english letter", tokenID)
+		return sdkerrors.Wrapf(ErrInvalidTokenID, "nft id(%s) only accepts lowecase alphanumeric characters, and begin with an english letter", tokenID)
```
Comment (leejw51crypto) [inline x/nft/types/validation.go:53]: lowercase?

**Pull request merged**